k9s-bin/lure.sh

```diff
@@ -0,0 +1,19 @@
+name=k9s
+version=0.26.6
+release=1
+desc='TUI for managing Kubernetes clusters and pods'
+homepage='https://github.com/derailed/k9s'
+architectures=('amd64')
+license=('APACHE')
```

Comment:
I think it would be a good idea to standardize license names by using their SPDX identifiers. You can refer to this list: https://spdx.org/licenses/. I'll add that to the docs and update my own scripts to follow it.

```suggestion
license=('Apache-2.0')
```